tests/foreman/api/test_product.py

```diff
@@ -8,9 +8,9 @@
 
 :CaseAutomation: Automated
 
-:CaseLevel: Acceptance
+:CaseLevel: Component
 
-:CaseComponent: Hosts-Content
+:CaseComponent: Content Management
```

Comment:
```suggestion
:CaseComponent: ContentManagement
```
remove space , please

Reply:
we are requested to "Remove all characters outside of [-_a-zA-Z] range."